09week/tictactoe/index.html

```diff
@@ -25,8 +25,8 @@
   <div id="announce-winner"></div>
   <button id="clear">Clear Board</button>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+  <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script> -->
   <script type="text/javascript" src="./tttscript.js"></script>
-  <script type="text/javascript" src="./jquery.mobile-events.min.js"></script>
+  <!-- <script type="text/javascript" src="./jquery.mobile-events.min.js"></script> -->
```

Comment:
I didn't need these to make it work in the simulator, but if the `jquery.mobile-events.min.js` library was working, it might need to come before your script.